data/mods/ssb/moves.ts

```diff
@@ -98,6 +98,39 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
 		type: "Rock",
 	},
 
+	// n10siT
+	"unbind": {
+		accuracy: 100,
+		basePower: 60,
+		category: "Special"
+		desc: "",
+		shortDesc: "",
+		name: "Unbind",
+		pp: 15,
+		priority: 0,
+		flags: {protect: 1},
+		onTryMove() {
+			this.attrLastMove('[still]');
+		},
+		onPrepareHit(target, source) {
+			this.add('-anim', source, 'Hyperspace Hole', source);
+			this.add('-anim', source, 'Hyperspace Fury', source);
+		},
+		onHit(target, pokemon, move) {
+			if (pokemon.baseSpecies.baseSpecies === 'Hoopa') {
+				pokemon.formeChange(pokemon.species.otherformes[0], this.effect, false, '[msg]');
```

Comment:
Hoeen review the code before merging cause I've used some "clever" stuff like this

Reply:
Why not just use Aura Wheel's code as a base?

Reply:
I forgot it exists lol
Reason i used this code is n10 mentioned meloetta